Resources/Private/Fusion/Backend/Root.fusion

```diff
@@ -193,7 +193,7 @@ backend = Neos.Fusion:Template {
             label = 'Neos.Neos:Main:content'
             icon = 'file'
             uri = Neos.Neos:NodeUri {
-                node = ${q(site).parent().children().get(0)}
+                node = ${sitesForMenu[0]}
```

Comment:
Won't it be enough to just put?:
`node = ${site}`

Reply:
Oh yeah, that's even more simple. Seems to work just fine as well.